download.py

```diff
@@ -3,10 +3,14 @@
 from subprocess import call
 import requests
 from requests.auth import HTTPBasicAuth
+import http.cookiejar
```

Comment:
maybe you can add comment what you said here
`for python 2 replace http.cookiejar with cookielib`

Because py27 is running out soon, its ok to have support for py3 only.